python/kfserving/kfserving/kfmodel.py

```diff
@@ -30,6 +32,17 @@ def __init__(self, name: str):
         self.ready = False
         self.predictor_host = None
         self.explainer_host = None
+        # The timeout matches what is set in generated Istio resorurces.
+        # We generally don't want things to time out at the request level here,
+        # timeouts should be handled elsewhere in the system.
+        self.timeout = 600
```

Comment:
I wasn't sure if this should be set to 600 or something even larger.

Reply:
what's the default timeout?

Reply:
Default is 20 seconds. The idea is to set this to the highest timeout in the stack so timeouts don't happen here.